digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -351,7 +351,7 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     Map("roadNumber" -> roadAddressProjectFormLine.roadNumber,
       "roadPartNumber" -> roadAddressProjectFormLine.roadNumber,
```

Comment:
good catch. For some reason idea did not mark that as not used for me. It also has       "roadPartId" -> roadAddressProjectFormLine.roadNumber,  which doesn't seem right when roadpart and roadpartid are different things  in reservedRoadPartToApi method.  Probably better to remove it and if really needed, rewrite it.